applications/GeoMechanicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -43,6 +44,10 @@ void AddCustomUtilitiesToPython(const pybind11::module& rModule)
     pybind11::class_<KratosExecute::CallBackFunctions>(rModule, "KratosExecuteCallBackFunctions")
         .def(pybind11::init<std::function<void(const char*)>, std::function<void(double)>,
                             std::function<void(const char*)>, std::function<bool()>>());
+
+    pybind11::class_<ProcessUtilities>(rModule, "ProcessUtilities")
+        .def_static("AddProcessesSubModelPartList", &ProcessUtilities::AddProcessesSubModelPartList,
+                    pybind11::arg("custom_settings"), pybind11::arg("solver_settings"));
```

Comment:
I think it'd be nice to follow similar naming as the implementation:
```suggestion
                    pybind11::arg("project_parameters"), pybind11::arg("solver_settings"));
```

Reply:
changed